ConnectionJustForToday/ConnectionJustForToday/Models/JFTReading.swift

```diff
@@ -6,30 +6,67 @@
 //
 
 import Foundation
+import CryptoKit
 
 // Model Object for Just For Today Meditation Reading
 
 class JFTReading {
     
+    enum Key {
+        static let date = "date"
+        static let title = "title"
+        static let pageNumber = "pageNumber"
+        static let quote = "quote"
+        static let reference = "reference"
+        static let body = "body"
+        static let affirmation = "affirmation"
+        static let copyright = "copyright"
+        static let postJFT = "postsJFT"
+        static let uuid = "uuid"
+    }
+    
     
     // MARK: - Properties
     var date: String
     var title: String
+    var pageNumber: String
     var quote: String
+    var reference: String
     var body: String
-    var afffirmation: String
+    var affirmation: String
+    var copyright: String
     var postsJFT: [JFTPost]?
     var uuid: String
     
-    init(date: String, title: String, quote: String, body: String, afffirmation: String, posts: [JFTPost] = [], uuid: String = UUID().uuidString) {
+    init(date: String, title: String, pageNumber: String, quote: String, reference: String, body: String, affirmation: String, copyright: String, posts: [JFTPost] = [], uuid: String = UUID().uuidString) {
```

Comment:
I think this UUID is going to cause the problem that we talked about, where posts are not visible across instances of the reading

Reply:
Is it okay to leave for now and work on the SHA Tomorrow?